roles/profile/templates/profile_deploy.yml.j2

```diff
@@ -0,0 +1,3 @@
+---
+cache_path: {{ profile_symfony_cache_path }}
```

Comment:
might want to rename the file (and thus the name of the generated file) to `profile.yml` or something like that (akin to `/etc/openconext/engineblock.ini`)